Python/other/Plot_Graph.py

```diff
@@ -0,0 +1,51 @@
+import matplotlib.pyplot as plt
+import math as mt
+import random as rand
+
+# X and Y axis co-ordinates
+xx = []
+yy = []
+
+# Function to be ploted i.e the value of y
+f = lambda x : 2*x*x+5*x+3
+
+# Diffrent function to try from
+
+#f = lambda x : x
+#f = lambda x : pow(x,2)
+#f = lambda x : pow(x,3)
+#f = lambda x : pow(x,4)
+#f = lambda x : abs(x)
+#f = lambda x : mt.sin(x)
+#f = lambda x : mt.cos(x)
+#f = lambda x : mt.tan(x)
+#f = lambda x : rand.randint(0,100)
+#f = lambda x : mt.sqrt(x)
+#f = lambda x: 1/x
+
+
+# Number of Y co-ordinates
+limit = 50
+
+# Calculation the X co-ordinated using the function declared above
+for i in range(-limit, limit):
```

Comment:
What if I try to plot a graph of a function of form ```f(x) = 1 / x^n```? I guess the loop won't be able to append the value of y coordinate when the value of x gets 0.

Reply:
that's why under the loop I have given an `if` statement. If the user chooses  `f(x)=1/x` function, then he has to modify the loop accordingly. Same is true for the trigonometric functions, the user has to modify the limits.

Reply:
Okay, so please add a proper NOTE regarding this to avoid any such confusion.

Reply:
Sure